### ISSUE TITLE
Save user's tagging effort from interactive mode

### ISSUE THREAD

**Issue by dbosk** [dbosk/examgen#7]
Since the user tags the questions selected in interactive mode, it would be good to save this work. So these tags from interactive mode should be put into the code. This way the questions are correctly tagged when the exam is later used as a questions database.

**Comment by dbosk** [comment]
The way this is implemented breaks due to #16. We must reimplement this by keeping track of the differences between the tag sets and then edit in line, this is to preserve structures like these:
```LaTeX
\question Describe the following
\begin{parts}
% tags: crypto
\part cryptography,
% tags: privacy
\part privacy in decentralized systems
\end{parts}
```
Probably we should also add an option (-m ?) for whether the interaction should modify the tagging.

**Comment by dbosk** [comment]
Actually we can discuss whether saving this re-tagging effort makes sense. Probably the questions are accurately tagged and the user wants to override the tagging temporarily.
